public/locales/en/app.ftl

```diff
@@ -0,0 +1,35 @@
+-product-name = Firefox Monitor
+
+layout-support = Support
+layout-feedback = Give Feedback
+layout-terms = Terms&nbsp;and&nbsp;Privacy
+layout-copyright = Portions of this content are &copy; 1998-2018 by individual mozilla.org contributors. <br />Content available under a <a {{> analytics/link_event eventLabel="Creative Commons"}} href="https://www.mozilla.org/foundation/licensing/website-content/" target="_blank" rel="noopener">Creative Commons license</a>.
+
+error-not-subscribed = This email address is not subscribed to {-product-name}.
+error-hibp-throttled = Too many connections to HIBP.
+error-hibp-connect = Error connecting to HIBP.
+error-hibp-load-breaches = Could not load breaches.
+
+hibp-notify-email-subject = {-product-name} Alert : Your account was involved in a breach.
+
+home-title = {-product-name}
+home-not-found = Page not found.
+
+oauth-invalid-session = Invalid session
+oauth-confirmed-title = {-product-name} : Subscribed
+
+scan-title = {-product-name} : Scan Results
+
+user-add-invalid-email = Invalid Email
+user-add-email-verify-subject = Verify your subscription to {-product-name}.
+user-add-title = {-product-name} : Confirm Email
+
+user-verify-token-error = Verification token is required.
+user-verify-email-report-subject = Your Firefox Monitor report
```

Comment:
nit: instead of "Firefox Monitor" we should probably use `{-product-name}` here.